### PR TITLE
Update reusable-apps.txt

### PR DESCRIPTION
**Pull request by geryogam**
This PR provides the following changes to the [Advanced tutorial](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html):

- use [the recommended parameters](https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml) for the `[build-system]` table in pyproject.toml.
- use [the recommended front-end build tool](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives) `python -m build` to build a distribution package instead of [the deprecated](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html) `python setup.py sdist`.

### CODE REVIEW

**Comment by nanis94** [comment]
master

**Comment by timgraham** [comment]
Part of this was incorporated in 247e25897b1. Could you rebase your PR?

File: docs/intro/reusable-apps.txt [hunk 1]
```diff
@@ -242,8 +242,8 @@ this. For a small app like polls, this process isn't too difficult.
 #. Only Python modules and packages are included in the package by default. To
    include additional files, we'll need to create a ``MANIFEST.in`` file. The
    setuptools docs referred to in the previous step discuss this file in more
-   detail. To include the templates, the ``README.rst`` and our ``LICENSE``
-   file, create a file ``django-polls/MANIFEST.in`` with the following
+   detail. To include the static files, the template files, the ``README.rst`` and ``LICENSE``
```
Comment (timgraham) [inline docs/intro/reusable-apps.txt:245]: please wrap line length
Also I think it could be a little more concise:
```
... the templates, static files, ``README.rst``, and ``LICENSE``, 
```
(It's a little confusing that this change isn't related, so probably it should be a separate commit.)

File: docs/intro/reusable-apps.txt [hunk 2]
```diff
@@ -264,9 +264,10 @@ this. For a small app like polls, this process isn't too difficult.
    you add some files to it. Many Django apps also provide their documentation
    online through sites like `readthedocs.org <https://readthedocs.org>`_.
 
-#. Try building your package with ``python setup.py sdist`` (run from inside
+#. Try building your package with ``python -m build`` (run from inside
```
Comment (timgraham) [inline docs/intro/reusable-apps.txt:267]: This requires `pip install build`, correct?

**Comment by felixxm** [comment]
Closing due to inactivity and lack of follow up.
